### ISSUE TITLE
Proposal: Add a new fullscreen option to ApplicationView.PreferredLaunchWindowingMode

### ISSUE THREAD

**Issue by benstevens48** [microsoft/microsoft-ui-xaml#3302]
Please let me know if I should post this elsewhere e.g. Project Reunion.

This is a relatively small request regarding launching an app in fullscreen mode. At the moment, you can set ApplicationView.PreferredLaunchWindowingMode to FullScreen to do this. However, there is one drawback in that it always launches on the main display of a multi-monitor setup, as opposed to the display which the app was last on, which is what all the other launch options do.

I don't propose changing this behavior in case people are relying on it, but instead I propose adding an additional option to `ApplicationViewWindowingMode`. This option would be called `FullScreenLastUsedDisplay` or something like that, and would launch the app in fullscreen on the last display on which the app was shown, like all the other options.

As a workaround, I currently switch to fullscreen during app launch using TryEnterFullscreen but it doesn't work 100% of the time.

(As an aside, it would be nice to have some more options to control which display the app is launched on. There could also perhaps be more control over which desktop a new view is launched on when using virtual desktops (i.e. current desktop vs the one on which the app is currently open).)

**Comment by StephenLPeters** [comment]
I don't think this is a part of winui, and that project reunion would be the better home for the issue. I've transferred it, but @MikeHillberg or @marb2000 to confirm.
